### DISCUSSION TITLE
GIT WIKI UPDATES

### DISCUSSION THREAD

**Discussion by joshvr27** [WebDevStudios/nextjs-wordpress-starter#364] [Ideas]
Hey guys,

Awesome write-up and repo!

Im following along with it now but wanted to add a few points you may want to include in the git wiki...

When pulling the wp-content repo i got a public key error, i managed to fix this by editing the line of code to clone via https instead of ssh.

When local is setup with the preferred PHP version the theme doesnt work (unsupported php version). 
I updated local to use PHP 8 and now all is fine n dandy!

Will keep posted as I work through the guide.

Awesome content once again, We use Headless as well but this is the first article Ive found that really dives deep into the many issues headless brings to the table.

**Comment by gregrickaby** [comment]
@joshvr27 I've updated the wiki with your suggestions. Thanks for the note and checking out the starter! 🍻 

https://github.com/WebDevStudios/nextjs-wordpress-starter/wiki/Backend-Setup
